extensions/shared/components/block-nudge/index.jsx

```diff
@@ -17,7 +17,8 @@ export const BlockNudge = ( { autosaveAndRedirect, buttonLabel, href, icon, subt
 					href={ href } // Only for server-side rendering, since onClick doesn't work there.
 					onClick={ autosaveAndRedirect }
 					target="_top"
-					isDefault
+					isSecondary
```

Comment:
This doesn't work, but I'm not sure why yet...

Reply:
It's working since I upgraded @wordpress/components

Reply:
`isSecondary` is a new prop that replaces `isDefault`.
Here more info: https://github.com/Automattic/jetpack/pull/14588#issuecomment-582884443